src/jllvm/object/Object.hpp

```diff
@@ -130,6 +130,11 @@ class Array : public AbstractArray
         return offsetof(Array<T>, m_trailing);
     }
 
+    constexpr operator llvm::ArrayRef<T>()
+    {
+        return {m_trailing, m_length};
+    }
+
```

Comment:
Is this dead code? I am also quite against making this implicit given the issues we've had with relocations while iterating over an `Array`. If needed I think it'd be best to give this an explicit name (`toArrayRef` e.g.)